service/mcs/persistent/src/main/java/eu/europeana/cloud/service/mcs/persistent/CassandraDataSetService.java

```diff
@@ -498,6 +498,9 @@ public void deleteDataSet(String providerId, String dataSetId)
     @Override
     public void updateProviderDatasetRepresentation(String globalId, String schema, String version, Revision revision)
```

Comment:
what about inserting to provider_dataset_representation. This insertion should only be allowed if the revision is published .Right?